Actionable button fixes

agliga opened the pull request:
## Description
* Fixed combobox actionable svg position by making the non actionable styles ignored in actionable (added adjacent rules)
* Added example for textbox dual icons

## References
#1129 

## Screenshots
<img width="1141" alt="Screen Shot 2020-06-18 at 2 38 32 PM" src="https://user-images.githubusercontent.com/1755269/85074469-70328000-b171-11ea-85b1-ac3273bf1a2d.png">
<img width="1129" alt="Screen Shot 2020-06-18 at 2 38 44 PM" src="https://user-images.githubusercontent.com/1755269/85074472-71fc4380-b171-11ea-83d3-97494a193f0a.png">

ianmcburnie commented on docs/_includes/common/textbox.html:
```diff
@@ -162,6 +162,38 @@ <h3 id="textbox-icon-actionable">Textbox with Actionable Icon</h3>
+
+    {% highlight html %}
+<span class="textbox textbox--icon-end">
+    <input class="textbox__control" type="text" placeholder="placeholder text" />
```
Icon is missing from the example code.

agliga replied:
Good catch, always forget that

agliga commented:
> Can you also check RTL? I believe we have stories for those now.

Done, I added RTL examples. They all worked except for actionable one, I added a case for that on storybook and added a fix for that.
